lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -77,6 +84,15 @@ export class CdnSiteHostingConstruct extends cdk.Construct {
             behaviors: [{ isDefaultBehavior: true }],
           },
         ],
+        errorConfigurations: props.isRoutedSpa
+          ? [
+              {
+                errorCode: 404,
+                responseCode: 200,
+                responsePagePath: `/${props.websiteIndexDocument}`,
```

Comment:
Is it possible to specify `websiteIndexDocument` with a leading slash? If so, what does it do?

Reply:
(I'm not thinking you need to handle all sorts of complex paths; more wondering if there's any defence required.)

Reply:
I just updated what was here before which was `/index.html` so I guess we could end up in a situation where someone sets `websiteIndexDocument` with a leading slash, I can add a guard for this.